PivotController/PivotController/Controllers/PivotController.cs

```diff
@@ -16,6 +16,7 @@ public class PivotController : Controller
         private IWebHostEnvironment _hostingEnvironment;
         private bool isRendered;
         private PivotEngine<DataSource.PivotViewData> PivotEngine = new PivotEngine<DataSource.PivotViewData>();
+        private ExcelExport excelExport = new ExcelExport();

```

Comment:
Add export events for document customization.

Reply:
Added the events in sample